### DISCUSSION TITLE
Nextjs and RTL/LTR

### DISCUSSION THREAD

**Discussion by 4tendev** [saadeghi/daisyui#2409] [General]
When i trying using great Daisyui for dynamic language web app in my NextJS project there is problem that i cant solve .
As it needs to config the RTL : true/false in tailwind config file to work correctly in RTL and LTR document i cant find way to change the direction mode in my next app .

**Comment by saadeghi** [comment]
Related discussion:  
https://github.com/saadeghi/daisyui/discussions/1928#discussioncomment-6099010
